cms-YYYY-validated-runs/code/vali_records.py

```diff
@@ -123,14 +166,17 @@ def main():
     # this_year = all_years[year]
 
     # this gets json from the api server
-    url = 'http://api-server-cms-release-info.app.cern.ch/years?year='+year+'&output=plain'
-    this_year = json.loads(requests.get(url).text.strip())
+    url = 'http://api-server-cms-release-info.app.cern.ch/years?year='+year+'&type='+runtype+'&output=plain'
+    this_json = json.loads(requests.get(url).text.strip())
     
-    for val in this_year["val_json"]:
+    recid=int(recid)
```

Comment:
For Python we are trying to standardise code formatting to the `black` source code formatter, which would give:

```python
recid = int(recid)
```

Would you be up to using "black" formatting style? (E.g. you can install it in VS Code to make the formatting automated.) If so I can change it here and install a CI  check to make sure we'll stick to black code formatting. 

(I.e. we do for REANA since a few years, and are happy with it. I started to use it also here in this repository, but not "required" so far.)

Reply:
Sure, I can do that